tests/framework/client/node.go

```diff
@@ -0,0 +1,134 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"reflect"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	v12 "k8s.io/client-go/kubernetes/typed/core/v1"
```

Comment:
this import path should be changed imo - v12 says nothing informative about the pkg.